server/views/global/palette.config.yml

```diff
@@ -33,6 +33,8 @@ context:
     hex: "#717171"
   - name: accessible-grey-2
     hex: "#8f8f8f"
+  - name: smoke
+    hex: "#e8e8e8"
```

Comment:
@alexwlchan Good point. We were initially generating the [cardigan palette](https://cardigan.wellcomecollection.org/components/detail/palette.html) by stuffing the colour names/hex values in a pseudo and reading them out with JS.

We didn't want to deploy non-app JS just in order to display the palette in the styleguide, but didn't succeed in copying the values across by another means, yet. I've added a `TODO:`